PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/build.gradle

```diff
@@ -23,10 +23,17 @@ dependencies {
     annotationProcessor 'org.projectlombok:lombok:1.18.8'
 
     compileOnly 'org.jetbrains:annotations:20.1.0'
+    implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-csv:2.11.2', {
+        exclude group: "com.fasterxml.jackson", module: "jackson-bom"
+    }
```

Comment:
let's use bom `group: "com.fasterxml.jackson", module: "jackson-bom"` to manage versions.

Reply:
Will resolve in later pr with other modules